AllReadyApp/Web-App/AllReady/Features/Events/UnregisterEventHandler.cs

```diff
@@ -1,27 +1,42 @@
 ﻿using System.Threading.Tasks;
+using System.Linq;
 using AllReady.Features.Notifications;
 using AllReady.Models;
 using MediatR;
 
 namespace AllReady.Features.Event
 {
-    public class UnregisterEventHandler : AsyncRequestHandler<UnregisterEvent>
-    {
-        private readonly IAllReadyDataAccess dataAccess;
+    public class UnregisterEventHandler : AsyncRequestHandler<UnregisterEvent> {
+        private readonly AllReadyContext dataContext;
```

Comment:
Personal niggle with the naming of the private instance fields. I personally prefer these are prefixed with the underscore. I believe this is the standard convention.